mantidproject/mantid
# Update to PEARL IDF

**Description of work.**

C. Ridley at ISIS provided an update to the PEARL IDF:

> I am looking to update the PEARL instrument definition file. There have been a couple of changes,` mainly to the finer details of where the monitors are positioned, and the exact length of the instrument flight path.
> 
> I have attached a working version, the dates at the top have been checked and are valid. I would like it to supersede the definition file shipped with Mantid called ‘PEARL_Definition_new_lowangle’.
> 
 

While performing the update I have taken the chance to update the file to use the `is=monitor` syntax rather than the old `mark-as` syntax to avoid spurious warnings on loading the file. My IDE also cleaned up the trailing whitespace. These final two changes are in separate commits to make it clear what the "real" updates were.

**To test:**

At ISIS load any PEARL file, PEARL00108392.nxs, and check that there are no longer any warnings about deprecated attributes. Show the detectors table and check the monitor distances match those in the diff.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
